doc/rados/operations/health-checks.rst

```diff
@@ -931,17 +939,18 @@ the system. Note that this marking ``out`` is normally done automatically if
 ``mgr/devicehealth/mark_out_threshold``).  If an OSD device is compromised but
 the OSD(s) on that device are still ``up``, recovery can be degraded.  In such
 cases it may be advantageous to forcibly stop the OSD daemon(s) in question so
-that recovery can proceed from surviving healthly OSDs.  This should only be
-done with extreme care so that data availability is not compromised.
+that recovery can proceed from surviving healthly OSDs.  This must be
+done with extreme care and attention to failure domains so that data availability
+is not compromised.
 
 To check device health, run the following command:
 
 .. prompt:: bash $
 
    ceph device info <device-id>
 
-Device life expectancy is set either by a prediction model that the Manager
-runs or by an external tool that is activated by running the following command:
+Device life expectancy is set either by a prediction model that the Ceph Manager
+runs or by an external tool by running the following command:
```

Comment:
I think removing the relative clause "that is activated by running the following command" makes this sentence ambiguous. Without the "that", the entire sentence falls under the scope of the prepositional phrase "by running the following command", but that prepositional phrase (I think, if I'm reading this correctly) applies only to the command that is used to designate an external tool as a part of the process of setting the life expectancy of the device. 

TL;DR: I think the sentence should have "that is activated" restored to it.

Reply:
I've pushed all of your most excellent suggestions except for this one.

My sense here is that an external tool may set a value for a given device's life expectancy by running `ceph device set-life-expectancy ...`. As originally written it implies to me that running `ceph device set-life-expectancy ...` _causes_ the external tool to be run / activated.

Reply:
I'll defer to you. I have no direct experience of this command, so I'm going to regard you as the authority.